zcash_client_backend/src/tor/http/cryptex/binance.rs

```diff
@@ -1,4 +1,3 @@
-use async_trait::async_trait;
```

Comment:
Can this also be used to remove librustzcash's one remaining dependency on `async_trait` for `data_api::chain::BlockCache`?

Reply:
Ah you're way ahead of me, lol. https://github.com/zcash/librustzcash/pull/1535#issuecomment-2439378806